posthog/settings.py

```diff
@@ -77,6 +77,7 @@ def get_list(text: str) -> List[str]:
 
 SELF_CAPTURE = get_from_env("SELF_CAPTURE", DEBUG, type_cast=str_to_bool)
 USE_PRECALCULATED_CH_COHORT_PEOPLE = not TEST
+CALCULATE_X_COHORTS_PARALLEL = get_from_env("CALCULATE_X_COHORTS_PARALLEL", 2)
```

Comment:
Isn't changing the name a breaking-ish / unexpected change for anyone who's been using `PARALLEL_COHORTS` env var?

Reply:
I highly doubt anyone has found this env var as it's undocumented, so I'm willing to take that risk to make the env var a bit clearer